index.rst

```diff
@@ -44,19 +47,20 @@ Constraints from FITS
 Because we need to be able to read subimages efficiently, file-level compression is not an option: our only options are FITS image "tile compression" and perhaps lesser-known FITS 4.0's binary table compression.
 Combined with our need to subimage reads over object stores, this already puts us beyond what third-party FITS libraries can do:
 
-- CFITSIO can do image tile compression, including efficient subimage reads that take advantage of the tile structure, but only on POSIX filesystems (or blocks of contiguous memory that correspond to the on-disk layout).
+- CFITSIO can do image tile compression and binary table compression, including efficient subimage reads that take advantage of the tile structure, but only on POSIX filesystems (or blocks of contiguous memory that correspond to the on-disk layout).
   It can also do subset reads of FITS binary tables, again only on POSIX filesystems.
+  CFITSIO can only permits customization of the quantization used in image tile compression via undocumented internal interfaces (which is what we use in ``afw``).
   This imposes the same limitation on its Python bindings in the ``fitsio`` package.
```

Comment:
This is not totally true, though I agree the documentation is sparse.

Reply:
We just added support for some of the lossy compression seeding options to fitsio: https://github.com/esheldon/fitsio/pull/411.

Excerpts from the docs are here in this github message: https://github.com/esheldon/fitsio/pull/411#issuecomment-2299028250